content-src/asrouter/components/SnippetBase/_SnippetBase.scss

```diff
@@ -36,7 +36,7 @@
 
     max-width: $wrapper-max-width-large;
     @media (min-width: $break-point-widest) {
-      max-width: $wrapper-max-width-widest;
```

Comment:
This change is incorrect, we want the width to change according to the inner content so it can be centred.

Also, `wrapper-max-width-widest` is correct for the spec according to aaron (legacy snippets did not have breakpoints/widths that aligned with new tab content)

Reply:
Ok I think we want `width` here instead so that they take up the required amount and still adjust on smaller screen.

With `width`
<img width="918" alt="grafik" src="https://user-images.githubusercontent.com/810040/48639361-42042180-e9cb-11e8-9a40-195037662029.png">

With `max-width`
<img width="857" alt="grafik" src="https://user-images.githubusercontent.com/810040/48639369-4cbeb680-e9cb-11e8-81d4-c6b5d7ad2975.png">

Reply:
I'm not seeing that problem with max-width, how can I reproduce that?

This is on nightly, the snippet seems to be aligned appropriately:

<img width="1127" alt="image" src="https://user-images.githubusercontent.com/1455535/48642211-2dfbf800-e9aa-11e8-91f2-3a5ea39798ac.png">